src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -787,6 +867,68 @@ discover_dfuse_mounts(void)
 	return rc;
 }
 
+static int
+fetch_dfs_cont_file_obj_with_fd(int fd, struct dfs_mt *dfs_mt, dfs_obj_t **obj)
```

Comment:
weird function name..

Reply:
How about fetch_dfs_file_obj_with_fd or fetch_file_obj_with_fuse? Any suggestions? Thank you!

Reply:
Or just fetch_dfs_obj_handle() as same as the one in il/int_posix.c?